sklearn/datasets/lfw.py

```diff
@@ -31,8 +31,8 @@
 
 from .base import get_data_home, _fetch_remote, RemoteFileMetadata
 from ..utils import Bunch
+from ..externals.pilutil import _imread, _imresize
```

Comment:
How fast is importing pillow/pil on your machine? I would keep the import local if it will slow accessing datasets significantly just because the user happens to have the library installed

Reply:
On my machine the import `from PIL import Image` takes approx. 0.04 seconds. The import `from sklearn.externals.pilutil import _imread, _imresize` takes 0.8 seconds though. Should I keep the global import or should I change it back to a local one?